javascript/javascript1/week4/js1week4hw.js

```diff
@@ -0,0 +1,118 @@
+"use strict"
+// Ex-Voice assistant
+// arry for do list
+const listToDo =[];
+//function that return a response of command
+let name = '';
+function getReply(command){
+    //Hello my name is Benjamin
+    let sentance = command.split (" ");
+    
+    if(command === 'Hello my name is Benjamin'){  
```

Comment:
What you did is right. This is an alternative method as well for example to convert to array of words
 let sentance = command.split (" ");

  if (sentance.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }